examples/reference/menus/SplitButton.ipynb

```diff
@@ -84,15 +84,15 @@
    "outputs": [],
    "source": [
     "split_button = pmui.SplitButton(items=[\n",
-    "    {'label': 'Download .parquet'},\n",
-    "    {'label': 'Download .xlsx'},\n",
-    "], label='Download .csv')\n",
+    "    {'label': 'Save'},\n",
+    "    {'label': 'Delete'},\n",
+    "], label='Save')\n",
```

Comment:
`width` and `sizing_mode="stretch_width"` not working

<img width="490" height="266" alt="image" src="https://github.com/user-attachments/assets/e3df52f2-c6b3-4429-9063-54dadd085749" />


```python
import panel as pn
import panel_material_ui as pmui

pn.extension()

pmui.Column(
    pmui.SplitButton(label="100px", width=100),
    pmui.SplitButton(label="200px", width=200),
    pmui.SplitButton(label="300px", width=300),
    pmui.SplitButton(label="stretch_width", sizing_mode="stretch_width"),
    sizing_mode="stretch_width",
).servable()
```